object-search/pods/vec.yml

```diff
@@ -0,0 +1,19 @@
+!NumpyIndexer
+with:
+  index_filename: vec.gz
+metas:
+  name: vec_idx  # a customized name
+  workspace: $WORKDIR
+requests:
+  on:
+    ControlRequest:
+      - !ControlReqDriver {}
+    SearchRequest:
+      - !VectorSearchDriver {}
+    IndexRequest:
+      - !ExcludeQL
```

Comment:
no need for this exclusion, since VectorIndexer will get only the embedding anyway

Reply:
This file was removed in favor of the compound indexer